functions-methods.js

```diff
@@ -33,4 +55,24 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+const emailOne = checkEmailValidity("n.eeken@novi.nl");
+const emailTwo = checkEmailValidity("n.eeken@novi.nl");
+const emailThree = checkEmailValidity("n.eekenanovi.nl");
+const emailFour = checkEmailValidity("n.eeken@novinl.");
+const emailFive = checkEmailValidity("tessmellink@novi,nl");
+
+function checkEmailValidity(emailAdres) {
+    const checkAtSign = emailAdres.includes("@");
+    const checkCome = emailAdres.includes(",");
+    const checkDot = emailAdres.lastIndexOf(".");
+    const checkDotAtEnd = checkDot !== emailAdres.length -1;
+
+    if (checkAtSign && !checkCome && checkDotAtEnd) {
+        return "True";
+    } else {
+        return "False";
+    }
```

Comment:
Je kan de uitkomst van de expressie meteen teruggeven:
```suggestion
    return checkAtSign && !checkCome && checkDotAtEnd
```